### PR TITLE
Create journal-week-2.md

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by nickycadavillo** [comment]
Great outline! Very informative! :+1:

**Comment by jeffworley** [comment]
Rosa!  This looks like an excellent start :+1:

**Comment by ShayHall** [comment]
:+1: Good start.  Probably should focus on 1 of those when you get to actually writing it for the sake of focus'.

File: journal-week-2.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+###Describe the MVC pattern of software design, as you understand it.
```
Comment (kellymurray) [inline journal-week-2.md:1]: like Shay suggested, you might want to focus on one for your final journal. :)

**Pull request merged**